### ISSUE TITLE
Does the Tiller Vault Plugin work with v2 HashiCorp k/v Vault?

### ISSUE THREAD

**Issue by brett-kingyens-mu** [markround/tiller#77]
Managed to use Tiller well with a Hashicorp v1 k/v Vault.
However we're finding challenges Tiller not converting secrets for a v2 k/v Vault.

On Hashicorp's website v2 Vaults need to add in 'data' to the path of the k/v path, if using their API.

Do we need to do the same for Tiller and adjust the value somewhere?  And if so, what is a working example?  (As we get 405 errors when trying different variations of inserting 'data' to the path, etc).

Or is it more that Tiller needs to have an update to adjust for any v2 k/v Vaults by Hashicorp?

**Comment by af6140** [comment]
Yeah, this is a good piece of software, hopefully the issue gets resolved.

**Comment by EugenMayer** [comment]
@markround should we consider tiller to be stalled / unmaintained in terms that you moved on to other areas? no offense at all, just a question for us to plan with. I open a dedicated issue for that so you can comment on it
